rewrite-yaml/src/main/java/org/openrewrite/yaml/MappingUtils.java

```diff
@@ -0,0 +1,9 @@
+package org.openrewrite.yaml;
+
+import org.openrewrite.yaml.tree.Yaml;
+
+public class MappingUtils {
```

Comment:
Would prefer not to pollute the package with a one method util class like this. Can we inline it for now?

Reply:
sure. i thought there might be more at the time but i removed them and only this was left.